ITEvents/Screens/Search tab/SearchPresenter.swift

```diff
@@ -40,9 +37,31 @@ class SearchPresenter: ISearchPresenter {
         if isListLayoutCurrent == settings.isListLayoutSelected { return }
         isListLayoutCurrent = settings.isListLayoutSelected
         view.toggleLayout(for: settings.isListLayoutSelected)
+        fetchEventsDebounced = debounce(
```

Comment:
```fetchEventsDebounced``` is not used in this PR at all.
But it probably should be used in `ISearchPresenter.searchEvents`
Each PR should be self-sufficient and complete feature.